src/main/java/io/github/lottetreg/httpserver/HTTPServer.java

```diff
@@ -1,13 +1,11 @@
 package io.github.lottetreg.httpserver;
 
-import java.io.PrintStream;
-
 public class HTTPServer {
   public static void main(String[] args) {
-    print(System.out);
-  }
-
-  public static void print(PrintStream out) {
-    out.println("Hello world!");
+    try {
+      new Server().start(new ServerSocket(new java.net.ServerSocket(5000)));
+    } catch (Exception e) {
+      e.printStackTrace();
```

Comment:
This is probably out of scope right now but this would be a great place to logging. Printing the stack trace seems fine for now.